src/main/java/frc/robot/subsystems/intake/factory/IntakeRollerRealConstant.java

```diff
@@ -0,0 +1,49 @@
+package frc.robot.subsystems.intake.factory;
+
+import com.revrobotics.SparkLimitSwitch;
+import edu.wpi.first.math.filter.Debouncer;
+import frc.robot.constants.IDs;
+import frc.robot.hardware.digitalinput.supplied.SuppliedDigitalInput;
+import frc.robot.hardware.motor.sparkmax.BrushlessSparkMAXMotor;
+import frc.robot.hardware.motor.sparkmax.SparkMaxWrapper;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import frc.robot.hardware.signal.supplied.SuppliedDoubleSignal;
+import frc.robot.subsystems.intake.IntakeRollerStuff;
+
+import java.util.function.BooleanSupplier;
+
+public class IntakeRollerRealConstant {
+
+	private final static double DEBOUNCE_TIME_SECONDS = 0.05;
+
+	private final static Debouncer.DebounceType DEBOUNCE_TYPE = Debouncer.DebounceType.kBoth;
+
+	private final static SparkLimitSwitch.Type REVERSE_LIMIT_SWITCH_TYPE = SparkLimitSwitch.Type.kNormallyOpen;
+
+	private final static double CONVERSION_FACTOR = 3.0;
```

Comment:
```suggestion
	private final static double GEAR_RATIO = 3.0;
```

I'm not sure though. @YHerm